examples/tut-6-rpc/Makefile

```diff
@@ -7,8 +7,14 @@ clean:
 server/build/%.main.mjs: server/%.rsh
 	$(REACH) compile $^ main
 
+# TODO: ditch this once `@reach-sh/rpc-client` package is available on NPM
+.PHONY: client-lib-js
+client-lib-js:
+	cd ../../js/rpc-client && npm pack
+	mv ../../js/rpc-client/reach-sh-rpc-client-0.0.1.tgz ./client-js
```

Comment:
`cp -fr`?

Reply:
Hmmm... Maybe I'm missing something obvious, but why is that preferable?

Reply:
No `npm` :)

Reply:
Ah, gotcha - thanks.

I'd actually done this without `npm pack` to begin with but I thought it kind of defeated the purpose based on what you said before WRT this being representative of how end users should build their own programs.

I'll tweak this next.

Reply:
How about 213d510 ?

Reply:
Coooooool